src/App/Commands/DbImportCommand.php

```diff
@@ -23,27 +24,32 @@ protected function configure()
         $this->setName('db-import')
             ->setDescription('Executes db import command')
             ->setHelp('Import given db into default database.')
-            ->addArgument('filepath', InputArgument::OPTIONAL, 'File to import');
+            ->addArgument('filepath', InputArgument::OPTIONAL, 'File to import')
+            ->addOption('dbname', NULL, InputOption::VALUE_OPTIONAL, 'The database in that we want to import the data');
     }
 
     protected function execute(InputInterface $input, OutputInterface $output)
     {
         $this->setupEnv();
 
         $filepath = $input->getArgument('filepath');
-
+        $databaseName = $_SERVER['MYSQL_DATABASE'];
+        if ($optionalDatabaseName = $input->getOption('dbname')) {
+          $databaseName = $optionalDatabaseName;
+        }
```

Comment:
`$databaseName = $input->getOption('dbname') ?? $_SERVER['MYSQL_DATABASE'];`

Please try before making the code change but I think this should work

Reply:
Thx, you right , that's better